Makefile

```diff
@@ -285,3 +303,26 @@ clean-coverage:
 
 clean:
 	$(RM) -r build
+
+$(TEST_DEPS_DIR)/$(PB_JAVA_JAR): Makefile
+	mkdir -p $(TEST_DEPS_DIR)
```

Comment:
Do you download it in a git-managed directory?

Reply:
`test/deps` contains only `opentelemetry-gen-classes.jar`, should I include this as well in the VCS?

Reply:
No. You should not mix VCS managed files and dependencies in the same directory, and you can't just include third-party binaries in the source repository without following license conditions of the respective project.

Reply:
Point taken, this is the tree now:
```
test/
├── deps
│   └── protobuf-java.jar
├── gen
│   └── opentelemetry-gen-classes.jar
```